frontend/src/utils/SerializerInputForm.svelte

```diff
@@ -172,8 +180,13 @@
   ) {
     unionTypeSelect = node["args"][0];
   }
-  if (node["type"] == "ofrak_type.range.Range") {
-    element = [];
+
+  $: if (node["type"].startsWith("ofrak")) {
+    setName();
+  }
+
+  if (node["type"].startsWith("ofrak")) {
+    element = [node["type"], {}];
```

Comment:
These feel redundant – `setName` essentially does the assignment that is replicated here.